app/components/chart-columns.tsx

```diff
@@ -84,7 +84,10 @@ export const ChartColumns = memo(
   }) => {
     return (
       <>
-        {fields.segment && fields.segment.type === "stacked" ? (
+        {/* FIXME: These checks should probably be handled somewhere else */}
+        {fields.segment?.componentIri &&
+        fields.segment?.componentIri !== FIELD_VALUE_NONE &&
```

Comment:
Just wondering: is this check even necessary? I would assume that when "none" is selected the whole `segment` field is removed from the config?